lib/source.js

```diff
@@ -75,27 +76,24 @@ module.exports = class CovSource {
   // given a start column and end column in absolute offsets within
   // a source file (0 - EOF), returns the relative line column positions.
   offsetToOriginalRelative (sourceMap, startCol, endCol) {
-    const lines = this.lines.filter((line, i) => {
-      return startCol <= line.endCol && endCol >= line.startCol
-    })
+    const lines = sliceRange(this.lines, startCol, endCol)
     if (!lines.length) return {}
 
     const start = originalPositionTryBoth(
       sourceMap,
       lines[0].line,
       Math.max(0, startCol - lines[0].startCol)
     )
+    if (!(start && start.source)) {
```

Comment:
I think we might not need to change this logic, if we figure out what's going on with the range detection logic.

Reply:
I rearranged those ifs to fail fast and avoid the costly originalEndPositionFor...but admittedly it's a nano-optimization...
it's all green now but I can remove it if you don't fancy it

Reply:
🤦 I actually just found out that the line coverage was tricked into 99.4%  by the execution of that originalEndPositionFor even if its result was not taken into account... 
now a part of me wants to put it back as it was while another feels that's a bit like cheating 😆